build/container.bzl

```diff
@@ -19,31 +19,27 @@ load(":platforms.bzl", "go_platform_constraint")
-# tarball output for the currently-configured architecture.
-# Additionally, if docker_push_tags is provided, uses multi_arch_container_push
-# to create container_bundles named push-NAME-ARCH with the provided push tags,
-# along with a push-NAME docker_push target.
-# Args:
-#   name: name used for the alias; the internal container_image and
-#     container_bundles are based on this name
-#   architectures: list of architectures (in GOARCH naming parlance) to
-#     configure
-#   base: base image to use for the containers. The format string {ARCH} will
-#     be replaced with the configured GOARCH.
-#   binary: the Bazel target name that builds the Go binary.
-#   docker_tags: list of docker tags to apply to the image. The format string
-#     {ARCH} will be replaced with the configured GOARCH; any stamping variables
-#     should be escaped, e.g. {{STABLE_MY_VAR}}.
-#   docker_push_tags: list of docker tags to apply to the image for pushing.
-#     The format string {ARCH} will be replaced with the configured GOARCH;
-#     any stamping variables should be escaped, e.g. {{STABLE_MY_VAR}}.
-#   tags: will be applied to all targets
-#   visibility: will be applied only to the container_bundles; the internal
-#     container_image is private
-#   All other args will be applied to the internal container_image.
+# arch-specific tagged container_bundles (named NAME-ARCH), an alias from NAME
+# to the appropriately NAME-ARCH container_bundle target, and a genrule for
+# NAME.tar copying the appropriate NAME-ARCH container bundle tarball output for
+# the currently-configured architecture. Additionally, if docker_push_tags is
+# provided, uses multi_arch_container_push to create container_bundles named
+# push-NAME-ARCH with the provided push tags, along with a push-NAME docker_push
+# target. Args: name: name used for the alias; the internal container_image and
+# container_bundles are based on this name architectures: list of architectures
+# (in GOARCH naming parlance) to configure base: base image to use for the
+# containers. The format string {ARCH} will be replaced with the configured
+# GOARCH. binary: the Bazel target name that builds the Go binary. docker_tags:
+# list of docker tags to apply to the image. The format string {ARCH} will be
+# replaced with the configured GOARCH; any stamping variables should be escaped,
+# e.g. {{STABLE_MY_VAR}}. docker_push_tags: list of docker tags to apply to the
+# image for pushing. The format string {ARCH} will be replaced with the
+# configured GOARCH; any stamping variables should be escaped, e.g.
+# {{STABLE_MY_VAR}}. tags: will be applied to all targets visibility: will be
+# applied only to the container_bundles; the internal container_image is private
+# All other args will be applied to the internal container_image.
+# TODO: remove kwargs, instead pass all args explicitly. I think we don't need
+# dynamic params and defining them explicitly would make this more clear.
```

Comment:
has this had some broken formatting applied? it looks far less readable, I'm guessing it's a mistake?

Reply:
Thanks for spotting that!
Line-wrapping incident 😄 
Should be fixed now